.ci-scripts/x86-64-unknown-linux-musl-release.bash

```diff
@@ -21,14 +19,15 @@ TRIPLE=${ARCH}-${VENDOR}-${OS}
 # Build parameters
 MAKE_PARALLELISM=8
 BUILD_PREFIX=$(mktemp -d)
-DESTINATION=${BUILD_PREFIX}/lib/pony
+PONY_VERSION=$(cat VERSION)
```

Comment:
I dont think this change should have been done. PONY_VERSION is a build parameter. We aren't using it. It's confusing. That isn't the version that Pony will show. Unlike in the nightly versions where it will. Can you switch back to no `PONY_VERSION` and having CLOUDSMITH_VERSION be `$(cat VERSION)` for the release entries?